src/commands/mquote.ts

```diff
@@ -0,0 +1,34 @@
+import { ICommand, PermissionLevel } from './base';
+import { Message, Client } from 'discord.js';
+import ServerSettingsRepository from '../repository/serverSettings';
+import WhiteListRepository from '../repository/whiteList';
+import TwitchClient from '../lib/twitch';
+import createMessageEmbed from '../wrapper/discord/messageEmbed';
+import { SetMutedPermissionsForChannel } from '../lib/mutedRole';
+import ObjectResolver from '../lib/objectResolver';
+import { CreateOrUpdateQuote } from '../lib/quote';
+
+export default class MQuoteCommand implements ICommand {
```

Comment:
Might want to make it ManualQuoteCommand as we're not that restricted to class name length and rather have better readability